Remove outdated image (#9430)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/campaigns/managing_campaigns/change_your_campaign_after_launch.md

2. Make your changes.
3. Select **Save as Draft**. Note that after creating a draft, you cannot edit the active campaign until you either launch or discard your draft.

![A draft of an active campaign with an option to view the active campaign.]({% image_buster /assets/img/campaign_draft.png %})

> You can reference the active campaign in the header of the campaign draft or the footer of the campaign analytics after selecting **View active Campaign**. To return to an active campaign, select **Edit Draft** from the analytics view or the active campaign view.

Reviewer: ```suggestion
You can reference the active campaign in the header of the campaign draft or the footer of the campaign analytics after selecting **View active Campaign**. To return to an active campaign, select **Edit Draft** from the analytics view or the active campaign view.
```
I'm a bit confused about what the first sentence is saying. What does it mean to reference an active campaign in a campaign draft or footer? 🤔

Author: Yeah! When you're in the draft of an active campaign, there's a banner across the top of the campaign editor with an option to view the active campaign (same for the footer in the analytics)

Reviewer: Oh!! Okay, that makes sense! I like your new edit! I was thinking that reference meant copying the link of the active campaign and pasting it into the header of the campaign draft 😅

> You can reference the active campaign in the header of the campaign draft or the footer of the campaign analytics after selecting **View active Campaign**. To return to an active campaign, select **Edit Draft** from the analytics view or the active campaign view.

Author: ```suggestion
As you're making edits to the draft, you can also reference the active campaign in the header of the campaign draft or the footer of the campaign analytics. 

To return to an active campaign, select **Edit Draft** from the analytics view or the active campaign view.
```
